contrib/webroot/404.html

```diff
@@ -101,7 +103,9 @@ <h5>OpenPGP Resources</h5>
           </p>
         </div>
         <div style="float:right;">
-          <p class="muted credit small">Provided as a public service by <a href="#" data-toggle="modal" data-target="#contact">###ENTERNAMEHERE###</a>.</p>
+          <!--
+              <p class="muted credit small">Use our Tor hidden service at <a href="${TOR_URL}">${TOR_URL}</a>.</p>
+          -->
```

Comment:
Suggestion for a followup, remove this, or replace with a template that emits this paragraph if the variable is set.

Reply:
Opened #305 for this. 👍🏻